app/code/Magento/Theme/view/frontend/layout/default.xml

```diff
@@ -8,6 +8,7 @@
 <page layout="3columns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <update handle="default_head_blocks"/>
     <body>
+        <attribute name="id" value="html-body"/>
```

Comment:
I don't see what exactly should be done in the task you mentioned - but adding an ID to the body is not really seems correct solution here.

Reply:
@ihor-sviziev We need to add this id to have a higher level of CSS specificity over less styles. This work is going in the scope of Page Builder story to eliminate the usage of inline styles. Feel free to ask me for any details.

Reply:
So why can't we use class here? I believe if you'll use `body.some-class .my-style` it should work approximately the same as `#my-body-id .my-style`

Reply:
Unfortunately, no. The class does not give enough specificity value. For more information, you could read https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity and visual specificity sheet https://specifishity.com/